src/spritekit.cs

```diff
@@ -1954,28 +1991,28 @@ partial interface SKAction : NSCoding, NSCopying {
 		
 		// These are in a category
 		[Static, Export ("moveByX:y:duration:")]
-		SKAction MoveBy (nfloat deltaX, nfloat deltaY, double sec);
+		SKAction MoveBy (nfloat deltaX, nfloat deltaY, double duration);
```

Comment:
It's a breakage change and it's less clear than (already less than perfect) `sec`, please keep the existing name.